crates/vision/src/field_color_detection.rs

```diff
@@ -1,13 +1,18 @@
 use color_eyre::Result;
+use serde::{Deserialize, Serialize};
+
 use context_attribute::context;
+use coordinate_systems::Isometry2;
 use framework::MainOutput;
-use nalgebra::Isometry2;
-use serde::{Deserialize, Serialize};
-use types::{field_color::FieldColor, interpolated::Interpolated};
+use types::{
+    coordinate_systems::{Field, Ground},
+    field_color::FieldColor,
+    interpolated::Interpolated,
+};
 
 #[derive(Deserialize, Serialize)]
 pub struct FieldColorDetection {
-    robot_to_field_of_home_after_coin_toss_before_second_half: Isometry2<f32>,
+    ground_to_field_of_home_after_coin_toss_before_second_half: Isometry2<Ground, Field>,
```

Comment:
:exploding_head:

Reply:
I'm all for removing the interpolation, but if we do want to add something like this again in the future, I think the correct way would be to introduce another coordinate system (`World` maybe) and then have a `field_to_world` transform which is either an identity or a 180 degree rotation.
Then we can do the interpolation in "world"-space.